connector_test.go

```diff
@@ -586,20 +555,13 @@ func testNetworking(t *testing.T, podmanNetworking string, containerTest string,
 		readBuffer = readOutputUntil(t, plugin, *mac)
 	}
 	logger.Infof(string(readBuffer))
-	if len(readBuffer) == 0 {
-		t.Fatalf("the container did not produce any output")
-	}
+	assert.Equals(t, len(readBuffer) > 0, true)
+
 	if ip == nil && mac == nil && expectedOutput != nil {
-		if strings.Contains(string(readBuffer), *expectedOutput) == false {
-			t.Fatalf("expected string not found: %s, %s found instead", *expectedOutput, string(readBuffer))
-		}
+		assert.Equals(t, strings.Contains(string(readBuffer), *expectedOutput), true)
 	} else {
-		if strings.Contains(string(readBuffer), *ip) == false {
-			t.Fatalf("expected string not found: %s, %s found instead", *ip, string(readBuffer))
-		}
-		if strings.Contains(string(readBuffer), *mac) == false {
-			t.Fatalf("expected string not found: %s, %s found instead", *mac, string(readBuffer))
-		}
+		assert.Equals(t, strings.Contains(string(readBuffer), *ip), true)
+		assert.Equals(t, strings.Contains(string(readBuffer), *mac), true)
```

Comment:
We do have a string contains assertion: https://github.com/arcalot/go-assert/blob/9911908433a59f9fecfbdc9cd872c8e26d0ebad9/equality.go#L10

Using equals instead removes helpful info from the error message.

Reply:
I think the information is expressed in the assertion statement, and printing the readBuffer to the log.